jenkins/internal-ci/generic/cortx-rgw-integration.groovy

```diff
@@ -0,0 +1,167 @@
+#!/usr/bin/env groovy
+pipeline {
+    agent {
+        node {
+            label "docker-${os_version}-node"
+        }
+    }
+
+    triggers {
+        pollSCM '*/5 * * * *'
+    }
+
+    environment {
+        version = "2.0.0"
+        env = "dev"
+        component = "cortx-rgw-integration"
+        release_dir = "/mnt/bigstorage/releases/cortx"
+        build_upload_dir = "$release_dir/components/github/$branch/$os_version/$env/$component"
```

Comment:
Can you please add a comment stating that few variables (e.g. `os_version`) are created as a manual parameters in jenkins job?
Note - This comment is for all groovy files in this PR

Reply:
Lets take this as an improvement as this need to be done in other files also.

Reply:
Fixed now